site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -48,7 +48,7 @@ curl -H 'Authorization: 2524a832-c1c6-4894-9125-41a9ea84e013'
 
 == JWT Authentication
 
-When an API is marked with a green identity icon such as &nbsp;icon:id-badge[role=green, title="Supports JWT",type=fas]&nbsp; it means you may call this API without
+When an API is marked with a blue identity icon such as &nbsp;icon:id-badge[role=green, title="Supports JWT",type=fas]&nbsp; it means you may call this API without
```

Comment:
good catch! It is confusing, the icon has a class of `green`, looks like we changed the color of the icon elsewhere and this class is not doing anything anymore.